sc/social/like/browser/configure.zcml

```diff
@@ -77,4 +77,11 @@
         permission="cmf.ManagePortal"
         />
 
+    <browser:page
+       for="*"
```

Comment:
register this view for site root only as the fallback image is the same all over the place; also, add a browser layer here.

Reply:
Ok